src/palladio/NodeParameter.cpp

```diff
@@ -135,8 +135,7 @@ int updateRPK(void* data, int, fpreal32 time, const PRM_Template*) {
 
 	// -- get style/name from start rule
 	auto getStartRuleComponents = [](const std::wstring& fqRule) -> std::pair<std::wstring, std::wstring> {
-		std::wstring style, name;
-		separate(fqRule, style, name, NameConversion::STYLE_SEPARATOR);
+		auto [style, name] = tokenizeFirst(fqRule, NameConversion::STYLE_SEPARATOR);
 		return {style, name};
 	};
```

Comment:
The entire lambda functions appears redundant now, since tokenizeFirst already returns a pair<wstring,wstring>

Reply:
good catch